airflow_munchkin/discovery_parser/models/method.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from typing import NamedTuple, List
+from airflow_munchkin.discovery_parser.models.parameter import Parameter, BODY_PARAM
+
+
+class Method(NamedTuple):
+    name: str
+    description: List[str]
+    params: List[Parameter]
+    """
+    :param name: name of the method ex. list, get, create
+    :param description: description of the method. Required in form of a list for textwrap.
+    :param params: parameters that could be passed to the method.
+    """
+
+    @classmethod
+    def from_dict(cls, method: dict):
```

Comment:
I would love to ruse `-> Method` however it will not work as the class is not yet defined. I could try using `-> "Method"` but I don't like this syntax.